worldedit-core/src/main/java/com/sk89q/worldedit/util/translation/LocalisationHelpers.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * WorldEdit, a Minecraft world manipulation toolkit
+ * Copyright (C) sk89q <http://www.sk89q.com>
+ * Copyright (C) WorldEdit team and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.sk89q.worldedit.util.translation;
+
+public class LocalisationHelpers {
+
+    /**
+     * Turn a translation key into a ".singular" or ".plural"
+     * depending on what the given number is.
+     *
+     * @param translationKey The base translation key
+     * @param number The number
+     * @return The key with .plural or .singular appended
+     */
+    public static String pluraliseI18n(String translationKey, float number) {
+        if (number == 1) {
```

Comment:
Can the if statement there may be changed? To `if (number == 1  || number == -1)`. As it should be eg. "-1 block".

Reply:
The only number that is singular is the number one. Any other number, including 0 and negatives are considered plural. I think the way it is right now is fine

Reply:
The only number that is singular is the number one. Any other number, including 0 and negatives are considered plural. I think the way it is right now is fine

Reply:
The only number that is singular is the number one. Any other number, including 0 and negatives are considered plural. I think the way it is right now is fine

Reply:
The only number that is singular is the number one. Any other number, including 0 and negatives are considered plural. I think the way it is right now is fine

Reply:
The only number that is singular is the number one. Any other number, including 0 and negatives are considered plural. I think the way it is right now is fine

Reply:
The only number that is singular is the number one. Any other number, including 0 and negatives are considered plural. I think the way it is right now is fine.

Reply:
The only number that is singular is the number one. Any other number, including 0 and negatives are considered plural. I think the way it is right now is fine.

Reply:
I'm just a random user, but I have an issue with this : in French -1, 0, 1, 1.5 and even 1.99 are all singular. Which means this system will not work correctly. 
This also breaks in Russian and most asian languages where the plural changes based on the number.

Reply:
Yes, this has been discussed before, but without a full localization lib to handle all the cases there will always be issues like this
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals

Reply:
A decision has been made to ignore all forms of pluralisation, similar to what Minecraft does. This prevents all of these corner cases.